source/postcard-schema/tests/schema.rs

```diff
@@ -239,12 +195,12 @@ fn owned_punning() {
 
     // TODO: This is wildly repetitive, and likely could benefit from interning of
     // repeated types, strings, etc.
-    assert_eq!(ser_borrowed_schema.len(), 268);
+    assert_eq!(ser_borrowed_schema.len(), 187);
```

Comment:
Serialized size is improved significantly by removing redundant names of primitives